README.md

```diff
@@ -76,6 +76,17 @@ pip list
 dbt deps
 ```
 
+## Usage
+### Quickrun
+Once you have set up your `.env` file, you can automatically run all of the below commands in sequence.  To do so, make sure the script has the correct permissions:
+```bash
+chmod 755 import.sh
+```
```

Comment:
I think since you ran this before pushing up `import.sh` it modified the file and no one will have to run `chmod 755 import.sh` again